src/Components/Body/Chat/LeftFold.tsx

```diff
@@ -0,0 +1,70 @@
+import { MdOutlineExpandMore } from "react-icons/md";
+import { IoFilterOutline } from "react-icons/io5";
+import { RiChatNewLine } from "react-icons/ri";
+import { RxTriangleDown, RxTriangleRight } from "react-icons/rx";
+import ChatBar from "./ChatBar";
+import { ChatListDataType } from "../../../Types/types";
+import { useState } from "react";
+type ChatLeftFoldProps = {
+  chatList: ChatListDataType[];
+  selectedChat: number;
+  changeSelectedChat: (num: number) => void
+}
+export default function ChatLeftFold({
+  chatList,
+  selectedChat,
+  changeSelectedChat
+}: ChatLeftFoldProps) {
+  const [showPinned,setShowPinned]= useState<boolean>(true)
+  const [showRecent,setShowRecent]= useState<boolean>(true)
+  return (
+    <div className="chatLeftFold">
+      <div className="chatListHeader">
+        <div className="listHeaderLeft">
+          <h2> Chat </h2>
+          <MdOutlineExpandMore />
+        </div>
+        <div className="listHeaderRight">
+          <IoFilterOutline />
+          <RiChatNewLine />
+        </div>
+      </div>
+      <div className="chat-total">
+        <div className="chatPinned">
+          <div className="chatpinheader" onClick={()=> setShowPinned(!showPinned)}>
+           {showPinned?<RxTriangleDown />: <RxTriangleRight/>}
+            <small> Pinned </small>
+          </div>
+          <div>
+            { showPinned &&
+              <ChatBar
+                key={chatList?.[0].id}
+                isActive={chatList?.[0].id === selectedChat}
+                chatData={chatList?.[0]}
+                changeSelectedChat={changeSelectedChat}
+              />
+           }
+          </div>
+        </div>
+        <div className="chatList">
+          <div className="chatpinheader" onClick={()=> setShowRecent(!showRecent)}>
+           {showRecent?<RxTriangleDown />: <RxTriangleRight/>}
+            <small> Recent </small>
+          </div>
+          <div>
```

Comment:
This part is same for both pinned and chats. We can create a wrapper component to handle the collapsible part.
This Collapsible component will take title and children in props.